client/webserver/site/src/js/register.ts

```diff
@@ -93,11 +93,11 @@ export default class RegistrationPage extends BasePage {
       const wallet = asset.wallet
       if (wallet) {
         const bondAsset = this.currentDEX.bondAssets[asset.symbol]
-        if (wallet.synced && wallet.balance.available > bondAsset.amount) {
+        const bondsFeeBuffer = await this.getBondsFeeBuffer(assetID, page.regAssetForm)
+        if (wallet.synced && wallet.balance.available > 2 * bondAsset.amount + bondsFeeBuffer) {
```

Comment:
Nice! I had just noticed this while doing some other work.

Reply:
Yeah, I totally missed this condition when I was rushing through https://github.com/decred/dcrdex/pull/2200

Reply:
Oh, the same thing exists on settings.ts.  Fixing that in another PR.  At least the first DEX uses the right info.

Reply:
The UI for bonds in settings is pretty rough. I was thinking about opening an issue. Would be good to see the bond sizes, fee buffer, durations, USD conversions, etc.

Reply:
100%